README.md

```diff
@@ -0,0 +1,152 @@
+
+```squirrel
+myPlot.setAxisTitles("Time", "Temperature (°F)");
+```
+
+## plot.addSecondYAxis(*axisTitle, traces [, callback]*)
+
+Adds a second y-axis on the right side of the graph and assigns the specified traces to it.  *traces* should be a list of the string names of traces as passed in the *traces* argument to the constructor.
+
+```squirrel
+myPlot.addSecondYAxis("Humidity (%)", ["inside_humidity", "outside_humidity"]);
+```
+
+## plot.setStyleDirectly(*styleTable [, callback]*)
+
+Sets the style of the graph by passing a description directly to the Plotly API.  This allows for advanced styling options that this library does not have specific methods for.
+
+*styleTable* should be a Squirrel list or table that will be parsed into JSON.  See the [Plotly API docs] (https://plot.ly/rest/) for details on how to format this argument.
+
+Note that there are several caveats to using this method:
+
+- This will entirely overwrite style parameters previously set using methods like `AddSecondAxis` or `setStyleDirectly`.
+- If there is an error in formatting *styleTable*, an error may be printed to the console or the call may silently fail.
+
+```squirrel
+local style =
+[
+    {
+        "name" : "temperature",
+        "type": "scatter",
+        "marker": {"symbol": "square", "color": "purple"}
+    },
+    {
+        "name" : "inside_humidity",
+        "type": "scatter",
+        "marker": {"symbol": "circle", "color": "red"}
+    }
+];
+myPlot.setStyleDirectly(style);
+```
+
+## Plotly.setLayoutDirectly(*layoutTable [,callback]*)
+
+See documentation for `setStyleDirectly`.
+
+## Plotly.plot(*dataObjs [, callback]*)
+
+Appends data to the Plotly graph.  This method takes an array *dataObjs* of Squirrel tables in the following form:
+
+```squirrel
+{
+    "name" : <TRACE_NAME>,
+    "x" : [<X_VALUE_1, X_VALUE_2, ...>],
+    "y" : [<Y_VALUE_1, Y_VALUE_2, ...>]
+    "z" : [<OPTIONAL_Z_VALUE_1>, <OPTIONAL_Z_VALUE_2>, ...]
+}
+```
+
+Note that the "x", "y", and "z" fields hold arrays of integers or strings and the "z" field is optional.
+
+Each element in *dataObjs* must have a name field that corresponds to a trace name as passed into the constructor.  To add multiple data points to a trace, either add them to the traces data arrays or make multiple calls to this method.
+
+```squirrel
+myPlot.post([
+    {
+        "name" : "temperature",
+        "x" : [timestamp],
+        "y" : [latest_temperature]
+    },
+    {
+        "name" : "inside_humidity",
+        "x" : [timestamp],
+        "y" : [latest_humidity]
+    }
+]);
+```
+
+## Static Methods
+
+## Plotly.getPlotlyTimestamp()
+
+Returns a timestamp string that Plotly will automatically recognize and style correctly.  Use this for your x-value on time-series data.
+
+```squirrel
+local timestamp = myPlot.getPlotlyTimestamp();
+myPlot.post(
```

Comment:
Since getPlotlyTimestamp is a static method, you probably want to document it's static usage (i.e. - call it as ```Plotly.getPlotlyTimestamp```, not ```plotlyInstance.getPlotlyTimestamp```